spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/repository/JdbcDataflowTaskExecutionMetadataDao.java

```diff
@@ -69,6 +69,10 @@ public class JdbcDataflowTaskExecutionMetadataDao implements DataflowTaskExecuti
 		"FROM %PREFIX%EXECUTION_METADATA M INNER JOIN " +
 		"%PREFIX%EXECUTION E ON M.TASK_EXECUTION_ID = E.TASK_EXECUTION_ID " +
 		"WHERE E.TASK_EXECUTION_ID = :taskExecutionId";
+	private static final String FIND_MANIFEST_BY_TASK_EXECUTION_IDS = "SELECT M.TASK_EXECUTION_MANIFEST AS TASK_EXECUTION_MANIFEST, M.TASK_EXECUTION_ID AS TASK_EXECUTION_ID " +
+		"FROM %PREFIX%EXECUTION_METADATA M INNER JOIN " +
```

Comment:
Just curious why we are doing an inner join on task execution, when it looks like we can use the task_execution_id from the manifest table.

Reply:
Good catch